CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 Nothing so far
 
+## 9.3.4 - 2021-11-17
+### Fixed
```

Comment:
Ik zou dit niet onder `Fixed` zetten, er is niets kapot, dit is eerder een `Changed` als ik iets moest kiezen uit het rijtje hierboven.

Reply:
:see_no_evil: Ah, goeie, dat zal ik zo aanpassen.